.github/workflows/zombienet.yml

```diff
@@ -0,0 +1,63 @@
+name: zombienet
+run-name: Zombienet tests
+on:
+  pull_request:
+
+jobs:
+  zombienet-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+          stable: true
+          check-latest: true
+
+      - name: Set cache variables
+        id: go-cache-paths
+        run: |
+          echo "::set-output name=go-build::$(go env GOCACHE)"
+          echo "::set-output name=go-mod::$(go env GOMODCACHE)"
+      - uses: actions/checkout@v3
+
+      - name: Go build cache
+        uses: actions/cache@v3
+        with:
+          path: ${{ steps.go-cache-paths.outputs.go-build }}
+          key: ${{ runner.os }}-go-build-${{ hashFiles('**/go.sum') }}
+          restore-keys: ${{ runner.os }}-go-build
+
+      - name: Go modules cache
+        uses: actions/cache@v3
+        with:
+          path: ${{ steps.go-cache-paths.outputs.go-mod }}
+          key: ${{ runner.os }}-go-mod-${{ hashFiles('**/go.sum') }}
+          restore-keys: ${{ runner.os }}-go-mod
+
+      - name: Run build
+        run: |
+          make build
+          echo "$HOME/work/gossamer/gossamer/bin" >> $GITHUB_PATH
+
+      - name: Install Zombienet
```

Comment:
nit do you think we could have a shared variable for the zombienet version v1.3.41?

Reply:
I don't understand what you mean by shared variable for...?

Reply:
Just one place to set the zombienet versions for the N references to it. Although not really necessary, I'll resolve this.